spec/unit/crypto.spec.ts

```diff
@@ -1010,18 +1010,24 @@ describe("Crypto", function() {
 
         it("encrypts and sends to devices", async () => {
             client.httpBackend
-                .when("PUT", "/sendToDevice/m.room.encrypted", {
-                    messages: {
-                        "@bob:example.org": {
-                            bobweb: encryptedPayload,
-                            bobmobile: encryptedPayload,
-                        },
-                        "@carol:example.org": {
-                            caroldesktop: encryptedPayload,
+                .when("PUT", "/sendToDevice/m.room.encrypted")
+                .check((request) => {
+                    const data = request.data;
+                    delete data.messages["@bob:example.org"]["bobweb"]["org.matrix.msgid"];
```

Comment:
Alternatively you can use `"org.matrix.msgid": expect.any(String)` in the `expect` statement